pkg/plugin/plugin.go

```diff
@@ -238,21 +235,6 @@ func (p *plugin) createOrUpdateExt(ctx context.Context, cs *api.OpenShiftManaged
 		return &api.PluginError{Err: err, Step: api.PluginStepClientCreation}
 	}
 
-	if cs.Properties.MonitorProfile.WorkspaceResourceID != "" {
-		authorizer, err := azureclient.GetAuthorizerFromContext(ctx, api.ContextKeyClientAuthorizer)
-		if err != nil {
-			return &api.PluginError{Err: err, Step: api.PluginStepClientCreation}
-		}
-		wc := operationalinsights.NewWorkspacesClient(ctx, p.log, cs.Properties.AzProfile.SubscriptionID, authorizer)
-		if err != nil {
-			return &api.PluginError{Err: err, Step: api.PluginStepClientCreation}
-		}
-		err = enrich.MonitorIDAndKey(ctx, wc, cs)
-		if err != nil {
-			return &api.PluginError{Err: err, Step: api.PluginStepEnrichMonitorProfile}
```

Comment:
This "api.PluginStepEnrichMonitorProfile" is not used anywhere else, we should remove it.

Reply:
Yes. However that doesn't block this PR.

Reply:
I think they should be removed here too as steps removal is happening here too